Add mirador (#383)
ratatui/awesome-ratatui: README.md

- [matchmaker](https://github.com/Squirreljetpack/matchmaker) - Fuzzy picker (FZF reboot).
- [md-tui](https://github.com/henriklovhaug/md-tui) - Markdown renderer in the terminal.
- [meteo-tui](https://github.com/16arpi/meteo-tui) - French weather app in the command line.

> - [mirador](https://github.com/jchultarsky/mirador) - A personal dashboard: world clocks, calendar and `.ics` agenda, weather, tasks, notes, a market watchlist and live CPU and network graphs, in a configurable grid.

Reviewer: ```suggestion
- [mirador](https://github.com/jchultarsky/mirador) - A personal dashboard with world clocks, calendar, weather, tasks, notes, a market watchlist and live CPU and network graphs.
```
